packages/Modal/styles.js

```diff
@@ -68,7 +68,7 @@ export const Dialog = styled(ReakitDialog)(
 )
 
 export const Content = styled(Box)`
-  padding: xxl;
+  padding: 50;
```

Comment:
todo for the v4, add 50 to new space '5xl'

Reply:
Yes please 😇